src/screens/WhotoFollow/index.js

```diff
@@ -141,21 +133,6 @@ const WhotoFollow = () => {
     setFollowed(copyFollowed);
   };
 
-  const onRefresh = React.useCallback(() => {
-    setRefreshing(true);
-    get({url: '/who-to-follow/list'})
-      .then((res) => {
-        setRefreshing(false);
-        if (res.status === 200) {
-          setUsers(res.data.body);
-        }
-      })
-      .catch((err) => {
-        crashlytics().recordError(new Error(err));
-        setRefreshing(false);
-      });
-  }, []);
-
   const register = async () => {
     setFetchRegister(true);
     Analytics.logEvent('onb_select_follows_btn_add', {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `onRefresh` function suggests that the developer has removed the pull-to-refresh functionality from the screen. If this functionality is not required, then its removal simplifies the code. However, if users expect this feature, its absence might negatively impact the user experience.
```diff
- const onRefresh = React.useCallback(() => {
-    setRefreshing(true);
-    get({url: '/who-to-follow/list'})
-      .then((res) => {
-        setRefreshing(false);
-        if (res.status === 200) {
-          setUsers(res.data.body);
-        }
-      })
-      .catch((err) => {
-        crashlytics().recordError(new Error(err));
-        setRefreshing(false);
-      });
-  }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->